test.py

```diff
@@ -29,6 +29,7 @@ def face_recognized(ctx):
 
 ctx = Context()
 ctx.add_module("ravestate_conio")
+ctx.add_module("ravestate_telegramio")
 # ctx.add_module("ravestate_tts_watson")
 # ctx.add_module("ravestate_facerec")
 ctx.add_module("hi")
```

Comment:
Not pertaining to you, but is the `"hi"` module there on purpose? @josephbirkner